src/routes/safe/components/Transactions/TxList/TxDetails.tsx

```diff
@@ -130,7 +131,7 @@ export const TxDetails = ({ transaction }: TxDetailsProps): ReactElement => {
       >
         <TxOwners txDetails={data} isPending={isPending} />
       </div>
-      {!isPending && !data.executedAt && txLocation !== 'history' && (canExecute || canCancel) && (
+      {!isPending && !data.executedAt && txLocation !== 'history' && !!currentUser && (
```

Comment:
Does it work correctly when you cannot confirm this tx? Like when you've already done it?

Reply:
Yes, they are only ever shown for queued transactions.

![image](https://user-images.githubusercontent.com/20442784/148241476-81137e34-e81b-4c62-8242-7573ce3019db.png)